docs/tutorials/clojure-guide.md

```diff
@@ -137,6 +137,38 @@ You may need to remove the lookup handlers conflict from `cider` and/or `clj-ref
 
 </details>
 
+<details>
+<summary>Spacemacs users</summary>
+
+- Add the `dap` layer to the `dotspacemacs/layers ()` section of your `.spacemacs` file (at the top of the file). 
+- Require the corresponding module in the `dotspacemacs/user-config ()` section of your `.spacemacs` (at the bottom of the file).
+(In my case, I made it work with Chromium.)
+- Make sure the Chromium executable is correctly setup (see https://github.com/syl20bnr/spacemacs/issues/13614#issuecomment-1179478365)
+ 
+
+
+```elisp
+dotspacemacs/user-config()
+  ...
+  ;; Enable DAP debugging in ClojureScript.
+  ;; Trying almost all options...
+
+  ;; This one downloads OK.
+  (require 'dap-chrome)
+
+  ;; These two cannot be downloaded... 
```

Comment:
these cannot be downloaded when you hit DOS protection of the marketplace

Reply:
Hmmm...

Have you considered using open-vsx.org[1] as a source instead of VS Marketplace? It's a libre alternative to VS Marketplace...
Plus, VS Marketplace can only be legally used by Microsoft products [2]...

[1] https://open-vsx.org/about
[2] https://github.com/VSCodium/vscodium/#extensions-and-the-marketplace

Reply:
@allentiak I was not aware of this option. Switching to it seems like a good idea